libs/utils/utils.js

```diff
@@ -109,7 +110,9 @@ const AUTO_BLOCKS = [
   { 'pdf-viewer': '.pdf', styles: false },
   { video: '.mp4' },
   { merch: '/tools/ost?' },
-  { 'mas-autoblock': 'mas.adobe.com/studio' },
+  { 'mas-autoblock': 'mas.adobe.com/studio', styles: false },
+  { m7: '/creativecloud/business-plans.html', styles: false },
+  { m7: '/creativecloud/education-plans.html', styles: false },
```

Comment:
Is it certain these links will always contain the `.html` part? Milo logic adds that suffix depending on the consumer's config, so some will have it, some will not. Is it a given that these particular links always contain it? L14 from `m7.js` also doesn't check for the `.html` suffix.

Reply:
Yes we are absolutely sure. These are external links 
https://www.adobe.com/creativecloud/business-plans.html
and
https://www.adobe.com/creativecloud/education-plans.html
If you remove the extension you will get 404.

Reply:
Just tested this redirection on prod page

<img width="1072" alt="Screenshot 2025-03-14 at 14 49 36" src="https://github.com/user-attachments/assets/740b41a8-d9f0-4ed5-9df9-4bd128c11b3d" />

Reply:
Great, thanks for the confirmation!